HW02/Yuliya_Bukhtik/yuliya_bukhtik_hw_02_t_02.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+file_path = 'log_file_task_02.txt'
+
+def task_2(file)
+  f = File.new file
+  lines = f.readlines
```

Comment:
You can do it in one line
```suggestion
  lines = File.readlines(file)
```